standalone_SCINGE.sh

```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+Data=$1
+outdir=$3
+gene_list=$2
+hypefile=hyperparameters.txt
```

Comment:
What is `hyperparameters.txt`?  Is that an example file we need to provide somewhere?  Also, will it be confusing if three arguments are read as bash script arguments but the hyperparameters file is hard-coded in the script?

Reply:
Yes, I forgot to include the hyperparameters.txt. There might be merit to hardcoding the hyperparameter file to ensure that the default method to run SINGE is what we would like it to be, but I agree with you that this could be confusing.
We could either have all of the above as command line inputs (could be quite confusing itself), or hard-code all inputs with recommendations on how to change them. Could we possibly keep a similar file which mimics SINGE_Example.m called testSINGEscript.sh?

Reply:
Some people expect that a shell script is like an executable, in which case it would take its own input arguments.  Other times, it is simply being used to track and pass input arguments to a different executable and record how that executable was run, in which case it is okay to hard code all inputs.

How do we envision this script?  Would it be reused by many different analyses, or would users copy it and keep a copy for each new analysis?